### ISSUE TITLE
Row rearrange

### ISSUE THREAD

**Issue by JaimeTak** [thegouger/MIDI2LR#12]
Thanks for the great work you're doing! My nanokontrol2 is working great. A suggestion: It would be nice to be able to rearrange the rows.

Thanks!

**Comment by rsjaffe** [comment]
Yes. Even being able to sort by clicking on the MIDI Command header would be nice. Ideally, for the LR Command column, it would be sorted like your command assignment popup is sorted (e.g., at the top would be Temperature, at the very bottom would be Adjustment Brush).

**Comment by rsjaffe** [comment]
This issue was moved to rsjaffe/MIDI2LR#6
